packages/browser/src/index.test.ts

```diff
@@ -93,12 +120,34 @@ describe('LogtoClient', () => {
     });
   });
 
-  describe('signIn', () => {
-    test('window.location should be correct signInUri', async () => {
+  describe('signIn, handleSignInCallback and isAuthenticated', () => {
+    beforeEach(() => {
+      sessionStorage.clear();
+    });
+
+    test('window.location should be correct signInUri just after calling signIn', async () => {
       const logtoClient = new LogtoClient({ endpoint, clientId }, requester);
       await logtoClient.signIn(redirectUri);
       expect(window.location.toString()).toEqual(mockedSignInUri);
     });
+
+    test('handleSignInCallback should throw LogtoClientError when the sign-in session does not exist', async () => {
+      const logtoClient = new LogtoClient({ endpoint, clientId }, requester);
+      await expect(logtoClient.handleSignInCallback(redirectUri)).rejects.toMatchError(
+        new LogtoClientError('sign_in_session.not_found')
+      );
+    });
+
+    test('should be authenticated after calling signIn and handleSignInCallback successfully', async () => {
+      const logtoClient = new LogtoClient({ endpoint, clientId }, requester);
+      await logtoClient.signIn(redirectUri);
+
+      const code = `code_value`;
+      const callbackUri = `${redirectUri}?code=${code}&state=${mockedState}&codeVerifier=${mockedCodeVerifier}`;
+
+      await expect(logtoClient.handleSignInCallback(callbackUri)).resolves.not.toThrow();
+      expect(logtoClient.isAuthenticated).toBeTruthy();
```

Comment:
should also check all the tokens have been set properly.

Reply:
Added the issues LOG-1245, LOG-1586 (add unit test cases for save and retrieve refresh token and id token in localStorage and object properties), and so on for adding these test cases later.

Since need to re-organize these test cases, intend to test tokens later in another PR.

Reply:
Updated. PTAL @simeng-li